index.rst

```diff
@@ -49,15 +49,15 @@ Platforms
    :caption: Platforms
    :maxdepth: 1
 
-   maps/index
-   sandbox/index
-   maps/geoportal
+   maps/deafrica_map
    maps/external_web_services
-   platform_tools/index
-   platform_tools/access
+   sandbox/index
    platform_tools/explorer
+   maps/geoportal
+   platform_tools/web_services_gis
    platform_tools/cube-in-a-box
-
+   platform_tools/odc-stac
```

Comment:
When I view the website build, I can see odc-stac listed in the body of the home page, but not in the left-hand menu bar. Can you please add odc-stac to the left-hand menu?